__tests__/components/AnalysisHeader/AnalysisHeader.test.tsx

```
@@ -77,4 +122,36 @@
       fireEvent.click(screen.getByTestId("create-analysis"));
     }).not.toThrow();
   });
+
+  it("should call onFiltersReset and update name when create analysis succeeds", () => {
+    const mockOnFiltersReset = jest.fn();
+    const mockGenerateAnalysisName = jest.fn(() => "New Analysis");
+
+    const useSequentialNamingMock = require("@/hooks/useSequentialNaming");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)